wiki/Help_Centre/de.md

```diff
@@ -0,0 +1,79 @@
+---
+tags:
+  - help
+  - issue
+  - problem
+  - trouble
+  - missing
+  - Hilfe
+  - Fehler
+  - Problem
+---
+
+# Hilfecenter
+
+Hast du bei irgendetwas Probleme? Wir sind hier, um zu helfen! Finde einige Lösungen zu alltäglichen Problemen in der Sitebar deines Bildschirms. Wenn dein Problem hier nicht auftaucht oder du immer noch keine Lösung hast, dann sende eine E-Mail zu [support@ppy.sh](mailto:support@ppy.sh). Alternativ kannst du dein Problem auch im [Hilfeforum](https://osu.ppy.sh/community/forums/5) schildern, in dem dir jemand weiterhilft.
+
+## Abschnitte
+
+Wähle den Abschnitt, der deinem Problem am meisten ähnelt, damit du eine passende Lösung findest.
+
+| Abschnitt | Schwerpunkte |
+| :-- | :-- |
+| [Account](/wiki/Help_Centre/Account) | osu!supporter, Einschränkungen, Einloggen, Namensänderungen, Profildaten |
```

Comment:
```suggestion
| [Account](/wiki/Help_Centre/Account) | osu!supporter, Sperrungen, Einloggen, Namensänderungen, Profildaten |
```
not sure if there is like a commonly used translation for restrictions, but this makes more sense i think